chaosinstana/probes.py

```diff
@@ -0,0 +1,51 @@
+from chaoslib.exceptions import ActivityFailed
+from chaoslib.types import Configuration, Secrets
+
+from logzero import logger
+
+import dateparser
+import datetime
+
+from .api import get_all_events
+
+__all__ = ["get_all_events_in_window", "convert_time_string_to_millis_string"]
+
+
+def convert_time_string_to_millis_string(source_time_string: str) -> str:
+    """
+    parse the time string to check its a valid time, first see if the
+    string is an int and treats as a milliseconds since epoch, if
+    the string is not an int uses dataparser
+    """
+    try:
+        source_time_int = int(source_time_string)
```

Comment:
I admit I don't understand what you are doing here :)

If source_time_string is already timestamp since epoch, why do you need to recompute it?

Reply:
dateparser can parse this by the way :)

```
dateparser.parse("1554734357")
```

If you need to make it UTC timezone-aware:

```
d = dateparser.parse("1554734357", settings={'RETURN_AS_TIMEZONE_AWARE': True, 'TO_TIMEZONE': 'UTC'})
```

Otherwise, you don't have that timezone information.